src/rebar_prv_common_test.erl

```diff
@@ -254,12 +254,12 @@ copy_and_compile_test_dirs(State, Opts) ->
         %% dir is a single directory
         Dir when is_list(Dir), is_integer(hd(Dir)) ->
             NewPath = copy(State, Dir),
-            [{dir, compile_dir(State, NewPath)}|lists:keydelete(dir, 1, Opts)];
+            [{dir, compile_dir(State, Dir, NewPath)}|lists:keydelete(dir, 1, Opts)];
         %% dir is a list of directories
         Dirs when is_list(Dirs) ->
             NewDirs = lists:map(fun(Dir) ->
                 NewPath = copy(State, Dir),
-                compile_dir(State, NewPath)
+                compile_dir(State, NewPath, NewPath)
```

Comment:
should be `compile_dir(State, Dir, NewPath)` no? to match what happens in the previous clause?